### ISSUE TITLE
Esperon-Rodriguez_0000: Processing submission

### ISSUE THREAD

**Issue by CaitlanB** [traitecoevo/austraits.build#245]
Data contributor: Manuel Esperon-Rodriguez
 
Esperon-Rodriguez_0000: only Data_Esperon_3.xlsx

Other data files:
Data_Esperon_1.xlsx: From various tree inventories
Data_Esperon_2.xlsx: Data from the Living Lab at Hawkesbury, Manuel plans to contribute more data on species and traits after 2020 summer
Data_Esperon_3.xlsx: Data from a paper currently in review. Will send paper after acceptance. 

NB Only a portion of data Manuel will eventually  send us 

- [x] Contact made with contributor (label as `waiting on response` if waiting for files)
- [x] Files received from contributor saved in google drive folder `austraits/data submissions/new`  ( label as `ready` )
- [x] Started importing files into own Austraits branch  (mark label as `In progress` and assign issue to relevant person )
- [x] Metadata complete, builds into Austraits
- [x] Report reviewed by AusTraits team
- [x] Report sent to contributor ( label as `waiting on response` )
- [x] Response received from contributor
- [x] Files updated & report finalised
- [x] Final checks ( all tests pass )
- [x] Branch merged into master branch (issue `closed` )

NB: 

- if you need to change the config files, e.g. to add a unit conversion or define a new trait. This should be done on the master branch (via pull or rebase)
